dev/examples/controls/line.js

```diff
@@ -490,7 +490,62 @@ export const renderLineCustomContainerPadding = (id) => {
     return lineDefault;
 };
 export const renderLineWithPanning = (id) => {
-    let graph;
+    const axisData = utils.deepClone(
+        getDemoData(`#${id}`, "LINE_TIMESERIES_DATELINE")
+    );
+    axisData.pan = {
+        enabled: true
+    };
+    const graphDataY = utils.deepClone(
+        getDemoData(`#${id}`, "LINE_TIMESERIES_DATELINE").data[0]
+    );
+    graphDataY.regions = [regions[0]];
+    const createGraph = () => {
+        graph.reflow();
```

Comment:
are we testing the dynamic data here for the reflow. in this example ?

Reply:
We are creating an example with dynamic data here [at line 548](https://github.com/cerner/carbon-graphs/pull/138/files#diff-8ef55c17e956a86bd177242c976fc979R548).